packages/amplify-category-function/src/provider-utils/awscloudformation/service-walkthroughs/lambda-walkthrough.ts

```diff
@@ -60,7 +60,7 @@ export async function createWalkthrough(
   // list out the advanced settings before asking whether to configure them
   context.print.info('');
   context.print.success('Available advanced settings:');
-  advancedSettingsList.forEach(setting => context.print.info('- '.concat(setting)));
+  advancedSettingsList.forEach(setting => context.print.info(setting));
```

Comment:
Why removed the '-' prefix? We rarely change prompts and DX without API reviews. Sync up on this with @renebrandel .

Reply:
The `-` served as the bullet-point in a pseudo unordered list. I replaced that with the icons in the PR. It results in a more visually comprehensible list of choices that can be understood better internationally.